spec/ddtrace/contrib/mongodb/client_spec.rb

```diff
@@ -465,7 +465,7 @@ def quantized?(object, options = {})
             expect(insert_span.resource).to match(/"operation"\s*=>\s*:insert/)
             expect(insert_span.status).to eq(1)
             expect(insert_span.get_tag('error.type')).to eq('Mongo::Monitoring::Event::CommandFailed')
-            expect(insert_span.get_tag('error.msg')).to eq('User  is not authorized to access test.')
+            expect(insert_span.get_tag('error.msg')).to match(/.*is not authorized to access.*/)
```

Comment:
This test now displays `"plain_user"` in the error message. I chose to make it less strict, to avoid flakiness in the future, in case MongoDB chooses to tweak their error string.